data-node/src/test/java/org/graylog/datanode/testinfra/DatanodeDevContainerBuilder.java

```diff
@@ -169,7 +169,7 @@ public GenericContainer<?> build() {
 
                 .withEnv("GRAYLOG_DATANODE_ROOT_USERNAME", rootUsername)
                 .withEnv("GRAYLOG_DATANODE_PASSWORD_SECRET", passwordSecret)
-                .withEnv("GRAYLOG_DATANODE_ROOT_PASSWORD_SHA2", rootPasswordSha2)
+                //.withEnv("GRAYLOG_DATANODE_ROOT_PASSWORD_SHA2", rootPasswordSha2)
```

Comment:
shouldn't we also remove the field and the builder method and its usages?

Reply:
Good point! Let me fix that.